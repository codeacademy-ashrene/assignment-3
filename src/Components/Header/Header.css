.bookShelf{
  display: flex;
  flex-direction: column;
  color: white;
}
.header{
  display: flex;
  flex-direction: row;
  width: 100%;
}

.headerLineLeft{
  width: 50%;
  float: left;
  align-self: center;
}
.headerLineRight{
  width: 50%;
  float: right;
  align-self: center;
}

.title{
  font-size: 30px;
  border: solid;
  border-radius: 40px;
  padding: 1%;
  margin-left: 5%;
  margin-right: 5%;
  background: none;
  color: white;
}
.subheading{
  font-style: italic;
  align-self: center;
  font-size: 20px;
}