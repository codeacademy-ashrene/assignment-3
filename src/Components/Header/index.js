import React, {Component} from 'react';
import './Header.css';

class Header extends Component {
  render(){
    return (
      <div className = "bookShelf">
      <div className = "header">
        <div className = "headerLineLeft"><hr/></div>
        <div className = "title">BS</div>
        <div className = "headerLineRight"><hr/></div>
      </div>
      <div className = "subheading">The Book Shelf</div>
      </div>
    );
  }
}

export default Header