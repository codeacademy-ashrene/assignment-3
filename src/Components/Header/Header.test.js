import React from 'react';
import renderer from 'react-test-renderer';
import Header from '.';


describe('Header',()=>{
    it('should render header',()=>{
        const tree=renderer.create(<Header  />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});