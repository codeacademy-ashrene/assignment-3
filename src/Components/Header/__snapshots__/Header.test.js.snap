// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render header 1`] = `
<div
  className="bookShelf"
>
  <div
    className="header"
  >
    <div
      className="headerLineLeft"
    >
      <hr />
    </div>
    <div
      className="title"
    >
      BS
    </div>
    <div
      className="headerLineRight"
    >
      <hr />
    </div>
  </div>
  <div
    className="subheading"
  >
    The Book Shelf
  </div>
</div>
`;
